File: FirstBit Workspace/C Programming/Assignments/Assignment 8/Q1.c
//Find minimum and maximum number in array.
#include<stdio.h>
void main(){
	
	int arr[10], max, min;
	
	//storing numbers in a array
	printf("Enter 10 numbers to store in a array: ");
	for(int i=0;i<10;i++)
		scanf("%d",&arr[i]);
		
	printf("\nYour array: ");
	for(int i=0;i<10;i++)
		printf("%d ",arr[i]);
		
	//finding max and min number from the array
	max=arr[0];
	min=arr[0];
	for(int i=0;i<10;i++){
		if(arr[i]>=max)
			max=arr[i];
		
		if(arr[i]<=min)
			min=arr[i];
	}
	
	printf("\nMaximum number from your array is: %d",max);
	printf("\nMinimum number from your array is: %d",min);
}

File: FirstBit Workspace/Core Java/In Class Work/ArrayCProgramsInJava/src/Q1.java
import java.util.Scanner;

public class Q1 {

	public static void main(String[] args) {
		
		//find minimum and maximum number in array
		
		//array reference 
		int[] array;
		
		//initialization
		array = new int[5];
		
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Enter elements to store in array: ");
		
		//store elements
		for(int i=0; i<array.length; i++) {
			array[i] = sc.nextInt(); 
		}
		
		System.out.println("\n Elements stored in array: ");
		//display array
		for(int ele:array) {
			System.out.println(ele);
		}
		
		int max = array[0], min = array[0];
		
		for(int ele:array) {
			if(ele>max) max = ele;
			if(ele<min) min = ele;
		}
		
		System.out.println("\nMaximum : " + max + "\nMinimum: " + min);
		
		sc.close();

	}

}
